Assets/RotateAround.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class RotateAround : MonoBehaviour {
```

Comment:
This name is way too generic.
If this is specially attached to the PlayerData or ice/fire.
Mark as MagicRotateAround or something.